libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -282,8 +282,14 @@ void AP_Frsky_Telem::send_SPort(void)
                 case SENSOR_ID_FAS: // Sensor ID  2
                     switch (_SPort.fas_call) {
                         case 0:
-                            send_sport_frame(SPORT_DATA_FRAME, DATA_ID_FUEL, (uint16_t)roundf(_battery.capacity_remaining_pct())); // send battery remaining
-                            break;
+                            {
+                                int8_t percentage;
+                                if (!_battery.capacity_remaining_pct(percentage)) {
```

Comment:
Not at all a deal breaker, but I wonder if whoever is in charge of the FrSky stuff knows if we should just not send the data if we don't have it?.

Either way I wouldn't make a change now, as it's clearly preserving the more important previous behavior.

Reply:
@badzz are you in charge of the FrSky stuff? Any thoughts?

Reply:
I believe @yaapu has been doing most of the work on FrSky